src/app/utils/isObject.test.ts

```diff
@@ -0,0 +1,16 @@
+import isObject from './isObject'
+
+describe('isObject', () => {
+  it('returns true', () => {
+    expect(isObject({ foo: 'bar' })).toBe(true)
+    expect(isObject({})).toBe(true)
+    expect(isObject([])).toBe(true)
+  })
+
+  it('returns false', () => {
+    expect(isObject(null)).toBe(false)
+    expect(isObject(Object.create(null))).toBe(false)
```

Comment:
This should be `true` instead, an object without a prototype is still an object.

```suggestion
    expect(isObject(Object.create(null))).toBe(true)
```

Reply:
Changed the condition back to the original and adjusted the tests accordingly